src/utils/utils.test.js

```diff
@@ -0,0 +1,11 @@
+import { formatDate } from './utils';
+
+describe('formatDate()', () => {
+    it('Should return correct date', () => {
+        expect(formatDate('2020-12-20')).toBe('20.12.2020');
+    });
+
+    it('Should return error', () => {
+        expect(() => formatDate('2020-20-202')).toThrowError(Error);
```

Comment:
Could you explain the purpose of this test?

Reply:
This test indicate that formatDate function should throw error when accept incorrect date string